package-scripts.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
     },
     test: {
       default: 'cross-env NODE_ENV=test jest --coverage',
+      update: 'cross-env NODE_ENV=test jest -u',
```

Comment:
Yeah, with `nps` it would be: `nps "test -u"` or `nps "t -u"` even. I'm working on making it so you don't need the quotes so you could do: `nps t -u` :) (you'd need `nps` installed globally or the local version available in your PATH, otherwise you'd have to do: `npm start t -u`).